.travis.yml

```diff
@@ -137,14 +137,14 @@ jobs:
       before_script:
         - docker build --tag=external-db docker/test-external-db
         - travis/create-new-project.sh
-        - travis/run-on.sh /tmp/travis/testgeomapfish/ ./docker-run travis/empty-make --makefile=travis.mk help
-        - travis/run-on.sh /tmp/travis/testgeomapfish/ ./docker-run make --makefile=travis.mk build
-        - travis/run-on.sh /tmp/travis/testgeomapfish/ docker-compose up -d
+        - travis/run-on /tmp/travis/testgeomapfish/ ./docker-run travis/empty-make --makefile=travis.mk help
```

Comment:
concerning travis/run-on, is not equal to cd ... && which could be easyier to read ?

Reply:
Patch welcome :-)